dotcom-rendering/src/components/FollowWrapper.importable.tsx

```diff
@@ -11,9 +13,17 @@ type Props = {
 };
 
 export const FollowWrapper = ({ id, displayName, format }: Props) => {
-	const [isFollowing, setIsFollowing] = useState<boolean | undefined>(
+	const [isFollowingNotifications, setIsFollowingNotifications] = useState<
+		boolean | undefined
+	>(undefined);
+	const [isFollowingTag, setIsFollowingTag] = useState<boolean | undefined>(
 		undefined,
 	);
+
+	const isMyGuardianEnabled = useIsMyGuardianEnabled();
+	const isBridgetCompatible = useIsBridgetCompatible('2.5.0');
+	const showFollowTagButton = isBridgetCompatible && isMyGuardianEnabled;
+
```

Comment:
I think we want `useIsMyGuardianEnabled()` `useIsBridgetCompatible()` run only on the client, so we should wrap them in a `useEffect` to make sure we don't try to run them when we server-side render the component.

With that in mind, `isMyGuardianEnabled`, `isBridgetCompatible` and `showFollowTagButton` might all need to become state so that we can trigger a re-render of the component on the client when they change.

Reply:
I think i fixed.` useIsMyGuardianEnabled()` and  `useIsBridgetCompatible()` are already hooks so I get an error if I wrape them in another useEffect in this component. I've assigned everything to a state hook in the component. I had to assign the hook results to variables to avoid a React Hook error : "useIsMyGuardianEnabled" is called conditionally. React Hooks must be called in the exact same order in every component render